components/ray/ray.go

```diff
@@ -69,6 +69,8 @@ func (r *Ray) ReconcileComponent(cli client.Client, owner metav1.Object, dscispe
 		}
 
 		if dscispec.DevFlags.ManifestsUri == "" || len(r.DevFlags.Manifests) == 0 {
+			// inject namespace for SCC
+			imageParamMap["ray-application-namespace"] = dscispec.ApplicationsNamespace
```

Comment:
Do we want to keep in specific to Ray, or open it to other component (and rename the param)?

Reply:
my first thought was to add it in a general way.
but then i am not sure, in case any of the component which already set the "namespace" in their params.env then will get overwritten by this function.
so i think the first step it to only add it for ray and name it to ray-application-namespace, then DW can use this variable in your SCC.
then we can have a follow up issue for all components, say we will use "namespace" variable as reserved variable by operator, see if any updates is needed? 
how you feel about this?

Reply:
I'd defer to your expertise, but I'd intuitively choose the generic approach. Even if a component already use the `namespace` variable, I think the intent would be that it'd match the namespace from the DSCI most of the time. If not, they could use a more specific variable name, that makes it clear, it's not the default installation namespace.

Reply:
did some updates on the logic